apps/portal/js/wizard.js

```diff
@@ -425,6 +437,9 @@ function getChartConfig(providerConfig) {
         contentType: "application/json",
         async: false,
         success: function (chartConfig) {
+            var chartConfig ={
```

Comment:
Lets facilitate the same fixes done for the provider config inorder to group the attributes for the chart config as well.